src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class name 'Drawable' sounds like a verb. Class and interface names should be nouns or noun phrases. Consider renaming it to something like 'DrawableFigure' if it represents a drawable figure.